test/service/test_request_validation.py

```diff
@@ -352,3 +327,20 @@ def test_order(self):
                     self.assertBadRequest(url, arg)
                 else:
                     self.assertResponseStatus(url, 200)
+
+    def test_version(self):
+        for endpoint in ('repository/files', 'fetch/repository/files'):
```

Comment:
```suggestion
        for endpoint in ['repository/files', 'fetch/repository/files']:
```

Reply:
Then again it would probably be less verbose to

```suggestion
        for fetch in [False, True]:
```

and